docs/demo/assets/styles/partials/_item.scss

```diff
@@ -55,14 +55,14 @@
 
 h2.f-item-heading {
 	@extend .dc-h2;
-	// text-transform:uppercase;
+	margin-bottom: $dc-space150;
 }
 
 h3.f-item-heading {
 	@extend .dc-h3;
 	text-transform:uppercase;
 	border-bottom:1px solid $dc-gray60;
-	// margin-bottom: $dc-space50;
+	margin-bottom: $dc-space100;
   padding-bottom: 0.4rem;
```

Comment:
Not sure if the linter is not picking this up, but I think we should try to keep the indentation consistent